### PR TITLE
Feature/style token selection piers

### PR DESCRIPTION
**Pull request by piersss**
Implementation of new design token selection

### CODE REVIEW

File: src/components/TextInput/TextInput.styles.tsx [hunk 1]
```diff
@@ -30,21 +29,17 @@ export const StyledTextInput = styled.div<TextInputStyleProps>`
...
+      color: ${(props) => props.theme.colors.lightGrey};
+    }
+
     &:focus {
       outline: 0;
-      border-bottom: 1px solid
-        ${(props) =>
-          props.hasError ? props.theme.colors.red : props.theme.colors.white};
     }
   }
```
Comment (piersss) [inline src/components/TextInput/TextInput.styles.tsx:44]: Let's keep the TextInput styling as minimal as possible.

File: src/components/TokenSelection/TokenSelection.tsx [hunk 2]
```diff
@@ -123,69 +143,99 @@ const TokenSelection = ({
...
-                : token.address === signerToken
-            } // shouldn't be able to select same duplicate token
-            removeActiveToken={removeActiveToken}
-            defaultToken={defaultActiveTokens[chainId!].includes(token.address)}
-            key={`${token.address}`}
-          />
-        ))}
+
+      <ScrollContainer ref={scrollContainerRef}>
+        <Legend>
```
Comment (gpxl-dev) [inline src/components/TokenSelection/TokenSelection.tsx:181]: IMO the legend should not scroll with the token list.

**Review by gpxl-dev — APPROVED** [review]
Implementation looks great. 

My comments are mostly opinions so aren't strong enough to reject. The only thing I think definitely needs changing is the button within a button problem, but there's currently no design on that so I am happy to merge with that issue for now.

Approving assuming styling is changed or not changed based on consensus, and button within button will be fixed at a later date.

File: src/components/TokenSelection/subcomponents/TokenDeleteButton/TokenDeleteButton.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+type TokenDeleteButtonProps = {
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+const TokenDeleteButton = ({ onClick }: TokenDeleteButtonProps) => {
+  const { t } = useTranslation(["orders"]);
+
+  return (
+    <Container onClick={onClick}>
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/components/TokenSelection/subcomponents/TokenDeleteButton/TokenDeleteButton.tsx:13]: Yeah this should be fixed later, there was no design for the delete button so I just left it there.

File: src/components/TokenSelection/TokenSelection.tsx (hunk 2 above)
Comment (piersss) [reply src/components/TokenSelection/TokenSelection.tsx:181]: But the legend is for the filteredTokens and not for inactiveTokens. This is something we can discuss with design.

File: src/style/GlobalStyle.ts [hunk 4]
```diff
@@ -4,6 +4,8 @@ import convertHexToRGBA from "../helpers/transformHexToRgba";
 import breakPoints from "./breakpoints";
 
 const GlobalStyle = createGlobalStyle`
+  @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700&display=swap');
```
[the remark this answers is not in the document]
Comment (piersss) [reply src/style/GlobalStyle.ts:7]: Yes I will implement this.

**Pull request merged**